doc/content/source/materials/ConcreteThermalMoisture.md

```diff
@@ -376,73 +396,101 @@ mechanisms separately. Instead, they tend to reproduce the general combined tren
 
 Calculation of $D_h$ starts with the calculation of a reference moisture diffusivity, $D_{h,0}$, at a given temperature, $T$, and relative humidity, $H$. Three reference moisture diffusivity $D_{h,0}$ models are implemented as:
 
-1. [!cite](mensi1988sechage)
+1. [!cite](xi_moisture_1994_a)
 
-!equation id=Dh0_mensi
-D_{h,0} = Ae^{BC}
+  [!cite](xi_moisture_1994_a) evaluate the moisture diffusivity for concrete according to
 
-where
+!equation id=Dh
+D_{h,0} = D_{Hcp}\left[1 + \frac{g_i}{\frac{1-g_i}{3} + \frac{1}{\frac{D_{Hagg}}{D_{Hcp}}-1}}\right],
 
-$D_{h,0}$ = humidity diffusion coefficient of concrete\\
-$A = 3.8 \times 10^{-13} m^2/s$ \\
-$B = 0.05$ \\
-$C$ = free water content in L/m$^3$
+  where
 
-$C$ is a function of relative humidity, $H$, in concrete as given by
+  $D_{h}$ = humidity diffusion coefficient of concrete \\
+  $D_{Hcp}$ = humidity diffusion coefficient of cement paste \\
+  $D_{Hagg}$ = humidity diffusion coefficient of aggregate (= 0 m$^2$/s) \\
+  $g_i$ = the volume fraction of aggregate
 
-!equation id=C_mensi
-C = HC_0
+  The humidity diffusion coefficient in cm$^2$/day for cement paste is expressed as [!cite](xi_moisture_1994_a):
+
+!equation id=Dhcp
+D_{Hcp} [m^2/s] = \alpha_h + \beta_h (1-2^{-10\gamma_h(H-1)}) * 1e-4 / 86400
+
+!equation id=alpha_h
+\alpha_h [cm^2/s] = 1.05 - 3.8 \frac{w}{c} + 3.56 \left(\frac{w}{c}\right)^2
 
-where $C_0$ is constant takes a value of 130 (in L/m$^3$).
+!equation id=beta_h
+\beta_h [cm^2/s] = -14.4 + 50.4 \frac{w}{c} - 41.8 \left(\frac{w}{c}\right)^2
+
+!equation id=gamma_h
+\gamma_h [cm^2/s] = 31.3 - 136 \frac{w}{c} + 162 \left(\frac{w}{c}\right)^2
+
+  where $\alpha_h$, $\beta_h$ and $\gamma_h$ are coefficients from test data. Since the value of the humidity diffusivity coefficient for aggregates, $D_{Hagg}$, typically
+  is negligible compared with the value of $D_{Hcp}$, it is assumed to be zero in the current implementation. Note that this model is applicable for concrete with w/c > 0.5.
+
+  Since no coupling between heat and moisture transfer is considered the value of $D_{ht}$, $C_{a}$, are $C_{w}$ considered as zero.
```

Comment:
No coupling between heat and moisture transfer is considered in this model, so the values of ... are set to zero.